chapters/part3/semantic-parsing-seq2seq.md

```diff
@@ -320,6 +320,29 @@ and it should give you the following numbers
 
 <exercise id="6" title="Decoding">
 
+Let us now look at what kinds of predictions the model makes. We'll use a `Predictor` to do so. Recall that we covered `Predictor`s in
+[an earlier chapter](/training-and-prediction#4). For our current purpose, we can directly use the
+[`Seq2SeqPredictor`](https://github.com/allenai/allennlp-models/blob/master/allennlp_models/generation/predictors/seq2seq.py) implemented in
+the `allennlp-models` repository, with the model archive we trained on our task.
+
+Let's try it now.
+
+<codeblock source="part3/semantic-parsing-seq2seq/predictor_source_easy" setup="part3/semantic-parsing-seq2seq/predictor_setup"></codeblock>
```

Comment:
When I run this I get something about downloading a spacy model.  Are you using a spacy tokenizer?  Can you avoid that, and just use a whitespace tokenizer?